docs/source/support/faq.rst

```diff
@@ -0,0 +1,183 @@
+==========================
+Frequently Asked Questions
+==========================
+
+What is the recommended specifications for running a nucypher?
+-------------------------------------------------------------------
+
+Worker nodes need to run ``nucypher`` and a local ethereum node, currently on the Goerli network. In total, you will
+require at 4GB for RAM. Nodes also need 24/7 uptime and a static, public IPv4 address.
+
+For ``nucypher`` specific requirements, see `System Requirements and Dependencies <https://docs.nucypher.com/en/latest/guides/installation_guide.html#system-requirements-and-dependencies/>`_.
+
+
+What is the network name for Incentivized Testnet?
+--------------------------------------------------
+
+The network name is ``cassandra``.
+
+
+How long is a period?
+---------------------
+
+1 period equals 24 hours. Periods begin at midnight UTC time.
+
+
+What's the difference between staker address and worker address? Can they be the same?
+--------------------------------------------------------------------------------------
+
+Technically, yes, but it is not recommended. The accounts have different security considerations - the staker address can be a high-value hardware wallet
+(staker with NU) that performs stake management while the worker is low-value and needs to remain
+unlocked while running (software wallet) since it used by an Ursula node.
+
+You should stake with one address and set the worker to be a different address Subsequently, you can bond
+the worker address to the stake.
+
+
+Is there a guide for Windows?
+-----------------------------
+
+Our guide is intended for Linux - we do not officially support Windows.
+
+
+Where is my Ursula config path?
+-------------------------------
+
+On Linux - ``$HOME/.local/share/nucypher/ursula.json``
+
+
+What is the difference between Standard Installation and Development Installation?
+----------------------------------------------------------------------------------
+
+The Development Installation is only needed for developing with ``nucypher``. You don't need to use
+it unless you plan to make changes to the codebase. If you are simply staking/running a node, you
+only need the Standard Installation
+
+
+How do I know that my node is setup correctly?
```

Comment:
I may be wrong, but I was under the impression that "setup" is a noun, and "set up" is a verb. Need a native speaker's opinion :)